saleor/dashboard/product/forms.py

```diff
@@ -2,28 +2,29 @@
 
 from django import forms
 from django.db import transaction
+from django.db.models import Count
 from django.forms.models import ModelChoiceIterator, inlineformset_factory
+from django.utils.encoding import smart_text
 from django.utils.translation import pgettext_lazy
 
 from ...product.models import (AttributeChoiceValue, Product, ProductAttribute,
                                ProductImage, ProductVariant, Stock,
-                               VariantImage, StockLocation)
+                               VariantImage, ProductClass, StockLocation)
 from .widgets import ImagePreviewWidget
 
-PRODUCT_CLASSES = {Product: 'Default'}
-
-
-class ProductClassForm(forms.Form):
-    product_cls = forms.ChoiceField(
-        label=pgettext_lazy('Product class form label', 'Product class'),
-        widget=forms.RadioSelect,
-        choices=[(cls.__name__, presentation) for cls, presentation in
-                 PRODUCT_CLASSES.items()])
 
+class ProductClassSelectorForm(forms.Form):
     def __init__(self, *args, **kwargs):
-        super(ProductClassForm, self).__init__(*args, **kwargs)
-        product_class = next(iter((PRODUCT_CLASSES)))
-        self.fields['product_cls'].initial = product_class.__name__
+        product_classes = kwargs.pop('product_classes', [])
+        super(ProductClassSelectorForm, self).__init__(*args, **kwargs)
+        choices = [(obj.pk, obj.name) for obj in product_classes]
+        if len(product_classes) > 5:
```

Comment:
This is the only field in this form. Not sure a dropdown enhances usability in any way.